target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -64,10 +72,16 @@ lantiq_setup_dsl()
 	avm,fritz3370-rev2-hynix|\
 	avm,fritz3370-rev2-micron|\
 	avm,fritz3390|\
+	avm,fritz3490|\
+	avm,fritz3490-micron|\
+	avm,fritz5490|\
```

Comment:
The Fritzbox 5490 does not have a DSL port.

Reply:
If will fix that error.